clusters/resource_cluster_test.go

```diff
@@ -1379,7 +1383,7 @@ func TestResourceClusterDelete_Error(t *testing.T) {
 		Fixtures: []qa.HTTPFixture{
 			{
 				Method:   "POST",
-				Resource: "/api/2.0/clusters/delete",
+				Resource: "/api/2.0/clusters/permanent-delete",
```

Comment:
What is this change?

Reply:
Earlier what we used to do is explicitly terminate the cluster and then call permanent delete (permanent delete automatically terminates the clusters so we don't need to do it explicitly now). This pathway is hit because this test is about returning the error while deletion happens which hits the delete (terminate) end point. As now we don't do it, we change it to permanent-delete.